articles/TOC.yml

```diff
@@ -6,6 +6,8 @@
     href: welcome.md
   - name: Five questions about quantum computing
     items:
+    - name: Overview
+      href: overview/index.md
```

Comment:
```suggestion

```

Reply:
PMs requested we remove this from TOC. Let's do it for now.

Reply:
Sure